imap/jmap_mailbox.c

```diff
@@ -1292,25 +1303,14 @@ static int _mboxquery_cb(const mbentry_t *mbentry, void *rock)
 {
     mboxquery_t *q = rock;
 
-    if (mbtype_isa(mbentry->mbtype) != MBTYPE_EMAIL)
-        return 0;
-
-    if ((mbentry->mbtype & MBTYPE_DELETED) && !q->include_tombstones)
+    if (!is_jmap_mailbox(mbentry, q->include_tombstones))
         return 0;
 
-    enum shared_mbox_type shared_mbtype = _shared_mbox_type(q->shared_mboxes, mbentry->name);
+    enum shared_mbox_type shared_mbtype = _shared_mbox_type(q->shared_mboxes, mbentry->name); 
```

Comment:
This line appears to be adding a trailing whitespace

Reply:
Removed